cms/static/js/spec/views/pages/container_spec.js

```diff
@@ -612,7 +653,9 @@ define(["jquery", "underscore", "underscore.string", "js/common_helpers/ajax_hel
                 page: PagedContainerPage,
                 initial: 'mock/mock-container-paged-xblock.underscore',
                 add_response: 'mock/mock-xblock-paged.underscore',
+                container: 'mock/mock-container-page.underscore',
                 has_visibility_editor: false
```

Comment:
Why is paged_specific_tests not specified to be true? And why is it not handled through the fixtures variable like has_visibility_editor and container?

One last question-- why is container now passed in as a variable (since it is the same for both)?